metrics/prometheus.go

```diff
@@ -1716,6 +1717,16 @@ func (c *PrometheusCollector) collectContainersInfo(ch chan<- prometheus.Metric)
 				}
 			}
 		}
+		if c.includedMetrics.Has(container.PerfMetrics) {
+			for _, metric := range stats.PerfStats {
+				labelNames := append(labels, "cpu")
```

Comment:
we should still add rawLabels here.

Reply:
I think that this is where `labels` and `values` [are coming from](https://github.com/google/cadvisor/blob/ba6fa06c73215e151a95a735a24e0e74d8da272a/metrics/prometheus.go#L1644).

Reply:
Oh, can we put this up in the table above instead of down here?

Reply:
`cpu` is perf specific - I would prefer to keep it local, to be honest.

Reply:
The table above allows specifying extra labels that are only applicable to the perf metric.

Reply:
Sorry, but I can't get what table you are referring to. Could you point me at line of code, please?

Reply:
https://github.com/google/cadvisor/blob/master/metrics/prometheus.go#L1474